examples/basics/ontologies.ipynb

```diff
@@ -43,7 +43,7 @@
     {
```

Comment:
<p>One more thing, could you please replace "instructions" with "name" since is the recommended way of doing it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
yep